lib/players/human.rb

```diff
@@ -0,0 +1,17 @@
+module Players
+  class Human < Player
+
+    def move(board, game)
```

Comment:
Thank you for your advice!
But I can't understand how to use "until" loop without local variable.
I'd like my program to keep asking until getting valid input.

My code is:
```
    def move(board, game)
      puts "Please select a number."
      puts "The upper left is '1', the bottom right is '9'."
      input = gets

      until board.valid_move?(input)
        puts "Please select a number."
        puts "The upper left is '1', the bottom right is '9'."
        input = gets
      end

      input
    end
```

And I tried following code, but it didn't work.
```
    def move(board, game)
      puts "Please select a number."
      puts "The upper left is '1', the bottom right is '9'."
      gets.chomp

      until board.valid_move?(gets.chomp)
        puts "Please select a number."
        puts "The upper left is '1', the bottom right is '9'."
        gets.chomp
      end

    end
```